app/screens/settings/notifications/notifications.tsx

```diff
@@ -25,6 +27,14 @@ const mentionTexts = {
         id: t('notification_settings.mentions_replies'),
         defaultMessage: 'Mentions and Replies',
     },
+    callsOn: {
+        id: t('notification_settings.calls_on'),
+        defaultMessage: 'On',
+    },
+    callsOff: {
+        id: t('notification_settings.calls_off'),
+        defaultMessage: 'Off',
+    },
```

Comment:
I'm inclined to change this whole const to use `defineMessages`

Reply:
definitely, thanks!